egret/parsers/rts_gmlc_parser.py

```diff
@@ -179,6 +179,10 @@ def create_model_data_dict(rts_gmlc_dir, begin_time, end_time, simulation="DAY_A
                 filtered_timeseries[name] = renewables_timeseries
 
     load_timeseries_spec = timeseries_pointer_dict[("Load",simulation)]
+    ## FIX issue with RTS-GMLC
```

Comment:
Since there is general activity / issue raising regarding how to ensure filesystem independence, shouldn't we switch any occurrence of "/" in code like this to os.sep?

Reply:
You are, of course, correct. I found on other instance of that in this file.

Since I was already using os.path.join I opted to just use that the proper way.